changelog.txt

```diff
@@ -1,3 +1,21 @@
+1.11.2 (2021-05-02)
+------------------------------------------------------------------------
+Change: [Win32] Limit hardware accelerated video driver to OpenGL 3.2 or higher (#9077)
+Change: More improvements to the GUI at different scales (#9075, #9102, #9107, #9133, #9174)
+Fix: [Network] Possible out-of-bounds memory access with certain password lengths (#9176)
```

Comment:
It's actually part of the salt used for company passwords where the out of bounds happens on. It has nothing to do with the actual length of the passwords.
What about: `Fix: [Network] Possible out-of-bounds memory access with certain lengths of server supplied company password salts (#9176)`